llvm/test/CodeGen/ARM/div-by-constant-to-mul-crash.ll

```diff
@@ -0,0 +1,56 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=arm--linux-gnueabihf -mcpu= -mattr=+neon | FileCheck %s
```

Comment:
I'm not sure what -mcpu= does, but can it be removed? And can you use armv7--linux-gnueabihf if it still shows the same issue.

Reply:
done